widget/text.go

```diff
@@ -348,6 +348,7 @@ func lineBounds(text []rune, wrap fyne.TextWrap, maxWidth int, measurer func([]r
 	if maxWidth == 0 || wrap == fyne.TextWrapOff {
 		return lines
 	}
+	maxWidth -= 2 * theme.Padding()
```

Comment:
Correct calculation, wrong location - could you move this to the call site? (line 92)

It is better for the caller to pass the unpadded width than hidden behaviour in this helper function because some callers may want padding, others may want double padding, and tests don't care about padding

Reply:
Done